feat(fastpass): subtitle suggesting full assessment in fast pass automated checks UI

dbjorge opened the pull request:
#### Description of changes

Adds subtitle help text under the "Automated Checks" title in the FastPass details view, per issue #385

![image](https://user-images.githubusercontent.com/376284/57252801-f63d4700-7001-11e9-9c0f-4ac58e5a1ba2.png)


#### Pull request checklist

- [x] Addresses an existing issue: Fixes #385
- [x] Added relevant unit test for your changes. (`npm run test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] Ran precheckin (`npm run precheckin`)
- [x] (UI changes only) Added screenshots/GIFs to description above
- [x] (UI changes only) Verified usability with NVDA/JAWS

dbjorge commented:
The code is ready to review, but marked "do not merge" because we want to wait til @cheade is in the office to review the final UI first.

ferBonnin commented:
@cheade could you take a look when you are back? 
Uses the same design we used to have for the old message :)

haonliu commented on src/tests/unit/tests/DetailsView/components/__snapshots__/adhoc-issues-test-view.test.tsx.snap:
```diff
@@ -1,6 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`AdhocIssuesTestView should return issues table with scanning to false 1`] = `"<IssuesTable deps={{...}} title=\\"test title\\" issuesTableHandler={{...}} issuesEnabled={true} issueTrackerPath={[undefined]} violations={{...}} issuesSelection={{...}} selectedIdToRuleResultMap={{...}} pageTitle={[undefined]} pageUrl={[undefined]} scanning={false} toggleClickHandler={[Function: clickHandlerS ...
+exports[`AdhocIssuesTestView should return issues table with scanning to false 1`] = `"<IssuesTable deps={{...}} title=\\"test title\\" subtitle={[undefined]} issuesTableHandler={{...}} issuesEnabled={true} issueTrackerPath={[undefined]} violations={{...}} issuesSelection={{...}} selectedIdToRuleResultMap={{...}} pageTitle={[undefined]} pageUrl={[undefined]} scanning={false} toggleClickHandler={[ ...
```
should we modify the test for adhoc-issues-test-view to make the subtitle change more clear in snapshots?

peterdur commented on src/DetailsView/components/issues-table.tsx:
```diff
@@ -27,6 +27,7 @@ export type IssuesTableDeps = IssuesDetailsPaneDeps & ExportDialogDeps;
 export interface IssuesTableProps {
     deps: IssuesTableDeps;
     title: string;
+    subtitle?: JSX.Element;
```
Does `subtitle` need to be optional? The code builds successfully without the `?`.

dbjorge replied:
That's because our codebase doesn't enforce strict null checks. I think it's more clear for it to be explicit about being optional, though (the visualizers for the non-issues cases intentionally don't set it)

cheade commented:
Hi all! 

I have some adjusted padding to this as a "subtitle" in terms of visual design, but in terms of content I just have some feedback :).

Starting a piece of info with "Did you know?" seems like it should be a part of a pop-up or a tip that's off to the side, not a permanent fixture to a page. "Did you know" can be a nice little grab for users, but it's typically not used in a permanent fixture. (I've seen Zoos that have separate signs from the main animal facts that are sometimes labeled "Did you know?" and contain fun facts).

**In terms of our voice**, I'm wondering if using the word _however_ is a bit to ponderous. If we do want to keep _however_ instead of something more informal like _but_, then I suggest we change up the sentence to one with a semi-colon instead of two. "Automated checks can detect some common accessibility problems, such as missing or invalid properties; however, most accessibility problems can only be identified through manual testing". 

**We can resolve this issue entirely with shortening the text**: "Did you know? Automated checks can only detect some common accessibility problems such as missing or invalid properties. The most reliable way to evaluate the accessibility compliance of a website or web app is to complete a full assessment."

Here is the [Figma link with the suggested padding and styling](https://www.figma.com/file/KXUMljdUYFveVJEPm91n8jNM/Accessibility-Insights-for-Web?node-id=2904%3A35858).

The reason why the text box is so short in length is to keep the character count in one single line close to 75. The ideal is between 45-75 for readability.

ferBonnin commented:
@cheade thanks for the suggestion on the text and the Figma. 
My only concern is that there is no explicit link between "Automated checks is not complete" and "do an assessment for checking accessibility compliance" (it feels like two separate things)

@LiLoDavis thoughts on the text suggestion?

LiLoDavis commented:
I agree that it seems clearer when we mention manual testing.  But I also agree the text could be shorter. And "Did you know?" looks a little odd immediately following the page title, so we could skip it. How about this?

Automated checks can detect some common accessibility problems such as missing or invalid properties. But most accessibility problems can only be discovered through manual testing, so the best way to evaluate web accessibility is to complete an assessment.

cheade commented:
I think this will work @LiLoDavis :). I've made just a small grammar tweak at the end, turning a compound sentence into two. @ferBonnin what do you think?

Automated checks can detect some common accessibility problems such as missing or invalid properties. But most accessibility problems can only be discovered through manual testing. The best way to evaluate web accessibility is to complete an assessment.

LiLoDavis commented:
@cheade, your edits look good to me.

ferBonnin commented:
@cheade looks good to me :)
@dbjorge could  you tweak the text, please?

dbjorge commented:
Updated styling and text per feedback. Updates screenshot (also updated in top PR description):

![image](https://user-images.githubusercontent.com/376284/57252834-0f45f800-7002-11e9-8bf8-98035ed598ef.png)

dbjorge commented:
PR is not retriggering for "retry all checks", closing and reopening PR to force a retry for what we think is a transient devops issue.
